setup.cfg

```diff
@@ -30,7 +30,7 @@ tests_require = pytest-astropy
 setup_requires = setuptools_scm
 install_requires =
     numpy>=1.17
-    pyerfa>=1.7
+    pyerfa>=1.7,<1.7.2
```

Comment:
Btw, we also have a `requires =` block above which I think is useless. But will keep that for another PR.
https://setuptools.readthedocs.io/en/latest/references/keywords.html
> requires is superseded by install_requires and should not be used anymore.